lib/brainstem/presenter.rb

```diff
@@ -53,11 +54,6 @@ def self.reflections(klass)
     end
 
     def self.ar_preload(models, association_names)
-      if Gem.loaded_specs['activerecord'].version >= Gem::Version.create('4.1')
```

Comment:
Looks like I forgot to yank the surrounding method somehow. This is injected in Preloader as a proc.